kubernetes/config/samples/classy-vision.yaml

```diff
@@ -0,0 +1,30 @@
+apiVersion: elastic.pytorch.org/v1alpha1
+kind: ElasticJob
+metadata:
+  name: elasticjob-sample
+spec:
+  rdzvEndpoint: "etcd-service:2379"
```

Comment:
This should be a template parameter?

Reply:
Agree. This sample comes with etcd.yaml. In that file, we use `etcd-service` as k8s service name. That's why we use etcd-service here.  I will change to placeholder